src/SourceBuild/content/eng/pipelines/pr.yml

```
@@ -48,3 +48,6 @@ stages:
       scope: lite
     ${{ else }}:
       scope: full
+
+- ${{ if eq(variables['isSourceOnlyBuild'], 'false') }}:
```

I think I would condition this on isSourceOnlyBuild not equals true:

```suggestion
- ${{ if ne(variables['isSourceOnlyBuild'], 'true') }}:
```